mysite/urls.py

```diff
@@ -0,0 +1,8 @@
+"""URL conf for mysite."""
+from django.contrib import admin
+from django.urls import path
+
+# pylint: disable=invalid-name
```

Comment:
This is a django-specific name which is required by the framework. It's going to be used in other urls.py files. So it makes sense to add it to the global list of good names (unless that pylint-django plugin already handles this case).